js/timers.js

```diff
@@ -79,14 +230,53 @@ function togglePanel() {
 
 window.addEventListener("DOMContentLoaded", function() {
     document.getElementById("timerConfigButton").addEventListener('click', togglePanel);
+    document.getElementById("closeConfigButton").addEventListener('click', togglePanel);
     document.getElementById("timerConfig").addEventListener("transitionend", hideConfigElements, true);
     var add_buttons = document.querySelectorAll("[data-action='add_timer']");
     for (i=0; i<add_buttons.length; ++i) {
         add_buttons[i].addEventListener("click", addTimer);
     }
     var clear_timers = document.querySelectorAll("[data-action='clear_timers']");
-    for (i=0; i<add_buttons.length; ++i) {
+    for (i=0; i<clear_timers.length; ++i) {
         clear_timers[i].addEventListener("click", removeAllTimers);
     }
 
-});
\ No newline at end of file
+});
+function startTimer(duration, display) {
+    var start = Date.now(),
+        diff,
+        minutes,
+        seconds;
+    function timer() {
+        // get the number of seconds that have elapsed since
+        // startTimer() was called
+        diff = duration - (((Date.now() - start) / 1000) | 0);
+
+        // does the same job as parseInt truncates the float
+        minutes = (diff / 60) | 0;
+        seconds = (diff % 60) | 0;
+
+        minutes = minutes < 10 ? "0" + minutes : minutes;
+        seconds = seconds < 10 ? "0" + seconds : seconds;
+
+        display.textContent = minutes + ":" + seconds;
```

Comment:
can we find a way to update the timer on the input here as well? Maybe something like these two lines

    display.parentNode.querySelectorAll("input.timerTimeMin").value = minutes;
    display.parentNode.querySelectorAll("input.timerTimeSec").value = seconds;